eng/common/SetupNugetSources.ps1

```diff
@@ -0,0 +1,160 @@
+# This file is a temporary workaround for internal builds to be able to restore from private AzDO feeds.
+# This file should be removed as part of this issue: https://github.com/dotnet/arcade/issues/4080
```

Comment:
https://github.com/dotnet/arcade/issues/3932 seems to be closed now?

Reply:
This file comes from arcade. Everything under eng/common is literally a copy of the latest arcade commit.

Should we instead open an issue in arcade asking that?

Reply:
Nah, if it's internal to arcade, I don't care about it that much given that most folks are not going to see this!